FindBounce/Kernel/FindBounce.m

```diff
@@ -1661,19 +1661,18 @@
 	dim = OptionValue["Dimension"]/.Except[3|4]:>(Message[FindBounce::dim];Return[$Failed,Module]);
 	maxIteR = OptionValue["MaxRadiusIterations"]/.Except[_Integer?Positive]:>(Message[FindBounce::posint,"MaxRadiusIterations"];Return[$Failed,Module]);
 	maxItePath = OptionValue["MaxPathIterations"]/.Except[_Integer?NonNegative]:>(Message[FindBounce::nonnegint,"MaxPathIterations"];Return[$Failed,Module]);
-	(* TODO: Decision if extension method is used should be taken in one place only.
```

Comment:
Sorry about this. I didn't understand the logic of how you decide if "extended" method is used or not.

Reply:
Hi Matevz, it is understandable., no problem 👍